spellbook/models/seaport/ethereum/seaport_ethereum_transfers.sql

```diff
@@ -0,0 +1,822 @@
+          '0x0000000000000000000000000000000000000000' then 'ETH'
+          when royalty_amount > 0 then t1.symbol
+          end as royalty_fee_currency_symbol
+          ,a.tx_hash || '-' || a.nft_token_id || '-' ||  concat('0x',substr(seller,3,40)) || '-' || 
+          order_type_id::string || '-' || cast(row_number () over (partition by tx_hash order by sub_idx) as 
+          string) as unique_trade_id,
+          a.zone
+      from p3_txn_level a
+          left join ethereum.transactions tx on tx.hash = a.tx_hash 
+                                              and tx.block_number > 14801608
+                                              {% if is_incremental() %} -- this filter will only be applied on an incremental run 
+                                              and tx.block_time >= (select max(block_time) from {{ this }}) 
+                                              {% endif %}
+          left join nft_ethereum.aggregators agg ON agg.contract_address = tx.to
+          left join tokens_ethereum.nft n on n.contract_address = nft_contract_address
+          left join tokens_ethereum.erc20 t1 on t1.contract_address = case when 
+          a.original_currency_contract = '0x0000000000000000000000000000000000000000' then 
+          '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+          else a.original_currency_contract end                                
+          left join prices.usd p1 on p1.contract_address = case when 
+          a.original_currency_contract = '0x0000000000000000000000000000000000000000' then 
+          '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+          else a.original_currency_contract end
+          and p1.minute = date_trunc('minute', a.block_time)
+          and p1.blockchain = 'ethereum')
+                                  
+,p4_call as (select 'match_orders' as main_type
+          ,'match_orders' as sub_type
+          ,idx as sub_idx
+          ,get_json_object(get_json_object(c.orders[0], "$.parameters"), "$.zone") as zone 
+          ,get_json_object(each, "$.offerer") as offerer 
+          ,get_json_object(get_json_object(each, "$.item"),"$.token") as offer_token 
+          ,get_json_object(get_json_object(each, "$.item"),"$.amount") as offer_amount 
+          ,get_json_object(get_json_object(each, "$.item"),"$.itemType") as offer_item_type 
+          ,get_json_object(get_json_object(each, "$.item"),"$.identifier") as offer_identifier 
+          ,get_json_object(get_json_object(each, "$.item"),"$.recipient") as recipient 
+          ,c.call_tx_hash as tx_hash
+          ,c.call_block_time as block_time
+          ,c.call_block_number as block_number
+          ,c.contract_address as exchange_contract_address
+     from (select *, posexplode(output_executions) as (idx, each) from {{ source('seaport_ethereum','Seaport_call_matchOrders') }}) c 
+     where call_success
+     {% if is_incremental() %} -- this filter will only be applied on an incremental run 
```

Comment:
similar to p2, this can be moved into subquery on line 641, but would also work as-is. just fits the overall pattern if moved